### PR TITLE
IGAPP-641: Move common module to web-ts

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web-ts/src/components/Selector.tsx [hunk 1]
```diff
@@ -88,9 +84,10 @@ const Selector = ({ items, activeItemCode, verticalLayout, closeDropDown, disabl
           return (
             <Element
               key={item.code}
-              as={Link}
+              // TODO As link
+              // as={Link}
+              // to={item.href}
```
Comment (Taggotty) [inline web-ts/src/components/Selector.tsx:89]: Why is this not yet working?

File: web-ts/src/components/__tests__/ScrollingSearchBox.spec.tsx [hunk 2]
```diff
@@ -56,6 +55,7 @@ describe('ScrollingSearchBox', () => {
     ).find(ScrollingSearchBox)
 
     component.find(SearchInput).prop('onFilterTextChange')('test')
+    // @ts-ignore
     component.find(Headroom).prop('onStickyTopChanged')(5)
```
Comment (Taggotty) [inline web-ts/src/components/__tests__/ScrollingSearchBox.spec.tsx:59]: Why ts-ignore?

**Review by Taggotty — APPROVED** [review]
Did you now use git move or not?
Some files actually preserved their history and some didn't. I personally don't really mind ^^

**Comment by klinzo** [comment]
> Did you now use git move or not?
> Some files actually preserved their history and some didn't. I personally don't really mind ^^

I did not use git move. I guess if there are too many changes, git does not recognize it as the same file anymore. I just did the same we did in native, first moving and renaming, commit and then changes and commit again.

File: web-ts/src/components/FailureSwitcher.tsx [hunk 3]
```diff
@@ -1,73 +1,71 @@
...
-  static renderContentNotFoundComponent(error: NotFoundError): React.Node {
+  static renderContentNotFoundComponent(error: NotFoundError): ReactNode {
     const { city, language } = error
     switch (error.type) {
       case 'category':
         return (
           <Failure
-            goToPath={new CategoriesRouteConfig().getRoutePath({ city, language })}
+            // TODO use right redirect path
+            // goToPath={new CategoriesRouteConfig().getRoutePath({ city, language })}
```
Comment (maxammann) [inline web-ts/src/components/FailureSwitcher.tsx:23]: Is there a ticket for this?

**Review by maxammann — APPROVED** [review]

File: web-ts/src/components/FailureSwitcher.tsx (hunk 3 above)
Comment (klinzo) [reply web-ts/src/components/FailureSwitcher.tsx:23]: Not yet but we have to check all todos afterwards anyway.

**Pull request merged**